documentation-site/examples/app-nav-bar/basic.js

```diff
@@ -9,7 +9,10 @@ import {
   Overflow as UserIcon,
   Upload as Icon,
 } from 'baseui/icon';
-import {Unstable_AppNavBar as AppNavBar} from 'baseui/app-nav-bar';
+import {
+  Unstable_AppNavBar as AppNavBar,
+  NAV_POSITION,
```

Comment:
should we just call this `POSITION`? historically, we never prefixed constants (like in the Button component: https://github.com/uber/baseweb/blob/master/src/button/constants.js)

Reply:
👍 I think in consts it can be `POSITION`.